src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,692 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * SquarelotronTest class.
+ */
+public class SquarelotronTest {
+    final static public int ZERO = 0;
+    final static public int ONE = 1;
+    final static public int TWO = 2;
+    final static public int THREE = 3;
+    final static public int FOUR = 4;
+    final static public int FIVE = 5;
+
+    /**********************************************************
+     **********************upsideDownFlip**********************
+     *********************************************************/
+
+    /**
+     * {{1, 2}, {3, 4}}.
+     */
+    @Test
+    public void testUpsideDownFlipTwoByTwoGridFirstRing() {
+        Squarelotron initialSquarelotron = new Squarelotron(TWO);
+        System.out.println("========INITIAL=========");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "========INITIAL=========" 36 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)